resources/helm/overlays/istio-telemetry/prometheus/templates/_sd_namespace.tpl

```diff
@@ -0,0 +1,8 @@
+{{- define "sdNamespaces" }}
+namespaces:
+  names:
+  - {{ .Values.global.istioNamespace }}
```

Comment:
This is still overwritten. I think it must be implemented in the reconciler.

Reply:
Added in mesh namespace in the updating code path.